simulation/sub8_gazebo/launch/duck.launch

```diff
@@ -2,7 +2,7 @@
   <param name="simulate" value="true" />
```

Comment:
That's an interesting thought, I can't think of any reason not to as long as everything is using it